config/initializers/carrierwave.rb

```diff
@@ -0,0 +1,14 @@
+CarrierWave.configure do |config|
+  if Rails.env.development? || Rails.env.test?
+    config.storage = :file
+  else
+    config.storage = :fog
+    config.fog_credentials = {
+      provider: 'AWS',
+      aws_access_key_id: ENV['AWS_ACCESS_KEY_ID']|| '',
+      aws_secret_access_key: ENV['AWS_SECRET_ACCESS_KEY'] || ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.